common/network-common/pom.xml

```diff
@@ -97,6 +97,11 @@
       <artifactId>slf4j-log4j12</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Can we move this dependency to L62 since the dependency around is test only. You can see the comment in L64 and L80